nschecker/notify.go

```diff
@@ -9,8 +9,6 @@ import (
 	"net/url"
 
 	"strings"
-
-	"github.com/nlopes/slack"
 )
 
 // Notifier interface is construct an response.
```

Comment:
it does not only construct an response.

It creates request and send them  to notification service.

```
// Notifier is interface of  notification
```
or something is fine